keyboards/kbdfans/kbd75/keymaps/ethan605/keymap.c

```diff
@@ -0,0 +1,62 @@
+    KC_LSFT,  MO(1),    KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RSFT,            KC_UP,    KC_PGDN,
+    KC_CAPS,  KC_LALT,  KC_LCMD,                      KC_SPC,   KC_SPC,   KC_SPC,                       KC_RCMD,  KC_RALT,  KC_RCTL,  KC_LEFT,  KC_DOWN,  KC_RGHT
+  ),
+
+  /* 1: Function layer
+   * ┌─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┐
+   * │     │     │     │     │     │     │     │     │     │     │     │     │     │     │     │ FN  │
+   * ├─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┴─────┼─────┤
+   * │     │     │     │     │     │     │     │     │     │     │     │     │     │   RESET   │     │
+   * ├─────┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬────────┼─────┤
+   * │        │ TOG │ MOD │ HU+ │ HU- │ SA+ │ SA- │ VA+ │ VA- │     │     │     │     │        │     │ // RGB controls
+   * ├────────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴────────┼─────┤
+   * │         │     │     │     │     │     │     │     │     │     │     │     │             │     │
+   * ├─────────┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴───────┬─────┼─────┤
+   * │            │     │     │     │ DEC │ TOG │ INC │STEP │     │     │     │          │     │     │ // Backlight controls
+   * ├──────┬─────┴─┬───┴──┬──┴─────┴─────┴─────┴─────┴─────┴────┬┴────┬┴────┬┴────┬─────┼─────┼─────┤
+   * │      │       │      │                                     │     │     │     │     │     │     │
+   * └──────┴───────┴──────┴─────────────────────────────────────┴─────┴─────┴─────┴─────┴─────┴─────┘
+   */
+  [1] = LAYOUT(
+    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  RESET,    _______,
+    _______,  RGB_TOG,  RGB_MOD,  RGB_HUI,  RGB_HUD,  RGB_SAI,  RGB_SAD,  RGB_VAI,  RGB_VAD,  _______,  _______,  _______,  _______,  _______,            _______,
+    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,                      _______,  _______,
+    _______,  _______,  _______,  _______,  BL_DEC,   BL_TOGG,  BL_INC,   BL_STEP,  _______,  _______,  _______,  _______,  _______,            _______,  _______,
+    _______,  _______,  _______,                      _______,  _______,  _______,                      _______,  _______,  _______,  _______,  _______,  _______
+  ),
+};
+
+void led_set_user(uint8_t usb_led) {
+    if (usb_led & (1 << USB_LED_CAPS_LOCK)) {
+        DDRB |= (1 << 2); PORTB &= ~(1 << 2);
+    } else {
+        DDRB &= ~(1 << 2); PORTB &= ~(1 << 2);
+    }
+}
```

Comment:
This function is deprecated.  Could you update to use this: 
https://docs.qmk.fm/#/custom_quantum_functions?id=example-led_update_user-implementation

Also for the pin controls, we recommend/encourage using the GPIO functions: 
https://docs.qmk.fm/#/internals_gpio_control

Reply:
Thanks @drashna, it seems that I don't need it anymore.